cmd/kubeadm/app/cmd/options/generic.go

```diff
@@ -19,11 +19,13 @@ package options
 import (
 	"github.com/spf13/pflag"
 	cliflag "k8s.io/component-base/cli/flag"
+	"k8s.io/kubernetes/cmd/kubeadm/app/constants"
 )
 
 // AddKubeConfigFlag adds the --kubeconfig flag to the given flagset
 func AddKubeConfigFlag(fs *pflag.FlagSet, kubeConfigFile *string) {
-	fs.StringVar(kubeConfigFile, KubeconfigPath, *kubeConfigFile, "The kubeconfig file to use when talking to the cluster. If the flag is not set, a set of standard locations are searched for an existing KubeConfig file.")
+	fs.StringVar(kubeConfigFile, KubeconfigPath, *kubeConfigFile, "The kubeconfig file to use when talking to the cluster. If the flag is not set, a set of standard locations can be searched for an existing kubeconfig file.")
+	fs.Lookup(KubeconfigPath).DefValue = constants.GetAdminKubeConfigPath()
```

Comment:
Though I consider this the right thing to do ATM, it is a bit hacky, so let's put a comment of some sorts here about a possible difference on help message default value and the actual default value.

Reply:
comment added.